services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.hashers import make_password
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.get(pk=user_id)
+    if username:
+        user.username = username
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    if password:
+        user.password = make_password(password)
+    user.save()
+
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)

```

Comment:
Consider handling the `DoesNotExist` exception that may be raised by `User.objects.get` to provide a more user-friendly error message or behavior.